camptix.php

```diff
@@ -1929,6 +1953,414 @@ function get_currencies() {
+				'format' => 'PYG %s',
+			),
+			'PEN'=> array(
+				'label' => __( 'Sol', 'camptix' ),
+				'format' => 'PEN %s',
+			),
+			'QAR'=> array(
+				'label' => __( 'Qatari Rial', 'camptix' ),
+				'format' => 'QAR %s',
+			),
+			'RON'=> array(
+				'label' => __( 'Romanian Leu', 'camptix' ),
+				'format' => 'RON %s',
+			),
+			'RUB'=> array(
+				'label' => __( 'Russian Ruble', 'camptix' ),
+				'format' => 'RUB %s',
+			),
+			'RWF'=> array(
+				'label' => __( 'Rwanda Franc', 'camptix' ),
+				'format' => 'RWF %s',
+			),
+			'SHP'=> array(
+				'label' => __( 'Saint Helena Pound', 'camptix' ),
+				'format' => 'SHP %s',
+			),
+			'WST'=> array(
+				'label' => __( 'Tala', 'camptix' ),
+				'format' => 'WST %s',
+			),
+			'SAR'=> array(
+				'label' => __( 'Saudi Riyal', 'camptix' ),
+				'format' => 'SAR %s',
+			),
+			'RSD'=> array(
+				'label' => __( 'Serbian Dinar', 'camptix' ),
+				'format' => 'RSD %s',
+			),
+			'SCR'=> array(
+				'label' => __( 'Seychelles Rupee', 'camptix' ),
+				'format' => 'SCR %s',
+			),
+			'SLL'=> array(
+				'label' => __( 'Leone', 'camptix' ),
+				'format' => 'SLL %s',
+			),
+			'SBD'=> array(
+				'label' => __( 'Solomon Islands Dollar', 'camptix' ),
+				'format' => 'SBD %s',
+			),
+			'SOS'=> array(
+				'label' => __( 'Somali Shilling', 'camptix' ),
+				'format' => 'SOS %s',
+			),
+			'LKR'=> array(
+				'label' => __( 'Sri Lanka Rupee', 'camptix' ),
+				'format' => 'LKR %s',
+			),
+			'SRD'=> array(
+				'label' => __( 'Surinam Dollar', 'camptix' ),
+				'format' => 'SRD %s',
+			),
+			'SZL'=> array(
+				'label' => __( 'Lilangeni', 'camptix' ),
+				'format' => 'SZL %s',
+			),
+			'TJS'=> array(
+				'label' => __( 'Somoni', 'camptix' ),
+				'format' => 'TJS %s',
+			),
+			'TZS'=> array(
+				'label' => __( 'Tanzanian Shilling', 'camptix' ),
+				'format' => 'TZS %s',
+			),
+			'TOP'=> array(
+				'label' => __( 'Pa’anga', 'camptix' ),
+				'format' => 'TOP %s',
+			),
+			'TTD'=> array(
+				'label' => __( 'Trinidad and Tobago Dollar', 'camptix' ),
+				'format' => 'TTD %s',
+			),
+			'UGX'=> array(
+				'label' => __( 'Uganda Shilling', 'camptix' ),
+				'format' => 'UGX %s',
+			),
+			'UAH'=> array(
+				'label' => __( 'Hryvnia', 'camptix' ),
+				'format' => 'UAH %s',
+			),
+			'UYU'=> array(
+				'label' => __( 'Peso Uruguayo', 'camptix' ),
+				'format' => 'UYU %s',
+			),
+			'UZS'=> array(
+				'label' => __( 'Uzbekistan Sum', 'camptix' ),
+				'format' => 'UZS %s',
+			),
+			'VUV'=> array(
+				'label' => __( 'Vatu', 'camptix' ),
+				'format' => 'VUV %s',
+			),
+			'VND'=> array(
+				'label' => __( 'Dong', 'camptix' ),
+				'format' => 'VND %s',
+			),
+			'YER'=> array(
+				'label' => __( 'Yemeni Rial', 'camptix' ),
+				'format' => 'YER %s',
+			),
+			'ZMW'=> array(
+				'label' => __( 'Zambian Kwacha', 'camptix' ),
+				'format' => 'ZMW %s',
+			),
+			'MRO' => array(
+				'label' => __( 'Mauritanian Ouguiya', 'camptix' ),
+				'format' => 'MRO %s',
+			),
```

Comment:
In the existing array of supported currencies, some have a `format` property, and some have a `locale`. I'm not sure when each is more appropriate. All of these additions have a `format`, but not a `locale`. Are we sure these are all correct? What is the source for the format data here?

Reply:
Hmm, I also a bit confused here. There are a couple of approaches that we can go ahead with: 

1. Use a same standard across all currencies, irrespective of how they are represented by that region. This answer makes a good point for going this way : https://ux.stackexchange.com/a/9109/47777

2. Figure out and add localization for each currency and use something like `money_format` (http://php.net/manual/en/function.money-format.php) to format the string. 

I want to go with 1, because then its more consistent. What do you think? Is there some other approach as well that I am missing on ?